sdk/identity/identity/samples/v3-beta/typescript/src/workloadIdentityCredential.ts

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+/**
+ * @summary Authenticates using Workload Identity Credential
+ */
+
+import { DefaultAzureCredential, WorkloadIdentityCredential } from "@azure/identity";
+import dotenv from "dotenv";
+
+dotenv.config();
+
+async function testDefaultCredential() {
```

Comment:
Not sure if we need the default credential test as part of Workload.  That was to ensure that if the environment variables were set, it was going to choose the workload identity.  Do you think it's still worthwhile to check?

Reply:
Yeah I think we should to test the integration of workload into DAC. We should probably keep it